Source/WebCore/inspector/WorkerInspectorController.cpp

```diff
@@ -99,11 +100,31 @@ void WorkerInspectorController::workerTerminating()
     m_debugger = nullptr;
 }
 
-void WorkerInspectorController::connectFrontend()
+void WorkerInspectorController::frontendInitialized()
+{
+#if ENABLE_REMOTE_INSPECTOR
+    if (m_pauseAfterInitialization) {
+        m_pauseAfterInitialization = false;
+        if (m_debuggerAgent) {
+            m_debuggerAgent->enable();
+            m_debuggerAgent->pause();
+        }
```

Comment:
I feel like this could be race-y. Given `JSGlobalObjectInspectorController` had to do some work to ensure its debugger agent always gets set up here, maybe I should do the same here (and introduce a similar `ensureDebuggerAgent()`).

Reply:
yeah you'd wanna force the `WorkerDebuggerAgent` to exist